playwright-e2e/tests/dsm/miscellaneous/permissions/gp-collection-date-permissions-check.spec.ts

```diff
@@ -0,0 +1,103 @@
+import { Locator, expect } from '@playwright/test';
+import { KitTypeEnum } from 'dsm/component/kitType/enums/kitType-enum';
+import { MainMenuEnum } from 'dsm/component/navigation/enums/mainMenu-enum';
+import { SamplesNavEnum } from 'dsm/component/navigation/enums/samplesNav-enum';
+import { StudyEnum } from 'dsm/component/navigation/enums/selectStudyNav-enum';
+import { Navigation } from 'dsm/component/navigation/navigation';
+import { KitsColumnsEnum } from 'dsm/pages/kitsInfo-pages/enums/kitsColumns-enum';
+import KitsSentPage from 'dsm/pages/kitsInfo-pages/kitsSentPage';
+import SearchPage, { SearchByField } from 'dsm/pages/samples/search-page';
+import Select from 'dss/component/select';
+import { testGPCollectionDate as test } from 'fixtures/dsm-fixture';
+
+test.describe('GP Collection Date Permissions Test', () => {
+  const studies = [StudyEnum.OSTEO2, StudyEnum.LMS];
+  const expectedKitTypes = [KitTypeEnum.BLOOD, KitTypeEnum.SALIVA];
+  const expectedAvailableMenuItems = [MainMenuEnum.SELECTED_STUDY, MainMenuEnum.SAMPLES];
+  const salivaKitType = KitTypeEnum.SALIVA; //Saliva kits are usually automatically created, so there'll be more of these to use for testing compared to blood kits
+  const expectedSampleMenuItems = [SamplesNavEnum.UNSENT_KITS_OVERVIEW, SamplesNavEnum.REPORT,
+    SamplesNavEnum.SUMMARY, SamplesNavEnum.KITS_WITHOUT_LABELS, SamplesNavEnum.QUEUE, SamplesNavEnum.ERROR,
+    SamplesNavEnum.INITIAL_SCAN, SamplesNavEnum.TRACKING_SCAN, SamplesNavEnum.FINAL_SCAN, SamplesNavEnum.RGP_FINAL_SCAN,
+    SamplesNavEnum.SENT, SamplesNavEnum.RECEIVED, SamplesNavEnum.SENT_RECEIVED_OVERVIEW,
+    SamplesNavEnum.SEARCH, SamplesNavEnum.LABEL_SETTINGS, SamplesNavEnum.CLINICAL_ORDERS];
+  let todaysKits: Locator[] = [];
+
+  for (const study of studies) {
+    test(`${study} - GP Collection Date Permissions Test`, async ({ page, request }) => {
+      const navigation = new Navigation(page, request);
+      const todaysMFBarcodes: string[] = [];
+      let currentKit: Locator;
+
+      await test.step('Create a user or use an existing one that only has kit_shipping permissions in pecgs [study group]', async () => {
+        //Select the study (either LMS or OS PE-CGS)
+        await new Select(page, { label: 'Select study' }).selectOption(`${study}`);
+
+        //Verify the current user can only see the navigation menu options allowed by the kit_shipping permission (just 'Selected study' + 'Samples' menu)
+        const availableNavigationMenuOptions = await navigation.getDisplayedMainMenu();
+        const amountOfMenuOptionsAvailable = availableNavigationMenuOptions.length;
+        expect(amountOfMenuOptionsAvailable, 'Study admin with only kit_shipping permission should only have 2 menu options').toBe(2);
+
+        expect(
+          availableNavigationMenuOptions,
+          `Displayed menu options do not match those expected of kit_shipping permission. Expected: 'Selected study' & 'Samples' only`).
```

Comment:
When `expect()` fails, does it emit the expected and actual values?  If so, then the error message probably shouldn't include the expected option because it might change, and then we'd need to remember to change the error message to.